<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarConductor</class>
 <widget class="QWidget" name="ModificarConductor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleModificarConductor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MODIFICAR CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIDConductor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bModificarConductor">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>MODIFICAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bRegresarConductor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>REGRESAR</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewMC">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>221</y>
     <width>570</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboxIDConductor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
